morph_tool/morphio_diff.py

```diff
@@ -2,8 +2,9 @@
 to see if two morphologies are the same or not'''
 import logging
 
-from numpy import allclose
+import numpy as np
 from morphio import Morphology
+from numpy import isclose
```

Comment:
Now the import of `isclose` is unused and can be removed. Did it pass the linting?

Reply:
LOL...
https://github.com/BlueBrain/morph-tool/pull/45